codalab/worker/worker_run_state.py

```diff
@@ -205,7 +223,19 @@ def mount_dependency(dependency, shared_file_system):
             #   dependency_path:docker_dependency_path:ro
             docker_dependencies.append((dependency.parent_path, dependency.docker_path))
 
+        log_bundle_transition(
```

Comment:
The function is called `_transition_from_PREPARING`. Why is `next_state` preparing?

Reply:
delete this code